src/algo/BruteForce.js

```diff
@@ -139,6 +144,37 @@ export default class BruteForce extends Algorithm {
 		this.implementAction(this.find.bind(this), text, pattern);
 	}
 
+	randomCallback() {
+		// The array indices correspond to each other
+		const textValues = [
+			'THIS IS A TEST TEXT',
+			'ABABABABABABABABABABABABAB',
+			'GGACTGA',
+			'BBBBAABBBAB',
+			'Machine Learning',
+			'Sphinx of black quartz, judge my vow',
```

Comment:
Similar issue as above but with vertical overflow here. It can be fixed by increasing the canvas height, but that's not ideal.